app/models/class_member.rb

```diff
@@ -12,6 +12,13 @@ class ClassMember < ApplicationRecord
 
   validate :student_has_the_school_student_role_for_the_school
 
+  has_paper_trail(
+    if: ->(cm) { cm.school_class&.school_id },
```

Comment:
just wondering if you need the `if` here, since it will always have an associated `school_class` (and school by extension of that), and if it didn't we'd want that change and any others audited I think?

Reply:
I didn't know if there would guaranteed always be a `school_class` just from unfamiliarity, so I'll remove this `if` :)

Reply:
Done in [a276520](https://github.com/RaspberryPiFoundation/editor-api/pull/431/commits/a276520fb9d2299218eb2163d94bcad6e862fc68) :)